tensorrt_llm/_torch/custom_ops/torch_custom_ops.py

```
@@ -27,14 +29,84 @@ def bmm_out(a: torch.Tensor, b: torch.Tensor, out: torch.Tensor) -> None:
+
+    def create_dummy_token_selected_experts(
+        inputs: List[torch.Tensor], ) -> List[torch.Tensor]:
+        input_tensor = inputs[0]  # First tensor is the input
+        # Generate dummy routing logits with correct shape
+        routing_logits_for_tuner = torch.randn(input_tensor.shape[0],
+                                               num_experts,
+                                               dtype=torch.bfloat16,
+                                               device=input_tensor.device)
+
+        # Apply routing to get properly shaped token_selected_experts
+        topk_ids_for_tuner, topk_weights_for_tuner = routing_method.apply(
```

Ruff (0.14.8)

Unpacked variable `topk_weights_for_tuner` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)